coderd/database/dbfake/databasefake.go

```diff
@@ -453,7 +453,7 @@ func (q *fakeQuerier) GetTemplateDAUs(_ context.Context, arg database.GetTemplat
 			continue
 		}
 
-		date := as.CreatedAt.UTC().Add(time.Duration(arg.TzOffset) * time.Hour).Truncate(time.Hour * 24)
+		date := as.CreatedAt.UTC().Add(time.Duration(arg.TzOffset) * time.Hour * -1).Truncate(time.Hour * 24)
```

Comment:
You are half right. 

I forgot to change it in the Dau dbfake function, which does break the unit test.

The template test does not break because I reverted the Template DAU to only use `UTC` again for performance reasons. So this is always `0` as the offset. 


----

tl;dr fixing it in the spot that is used **does break the test**